### ISSUE TITLE
Disappearing props

### ISSUE THREAD

**Issue by Kevrawr930** [Bouncyrock/TaleSpire-Beta-Public-Issue-Tracker#44]
**Describe the bug**
I was making a mock up tavern to test the building features and I noticed that after some time, some of the random props I had placed on other props or surfaces had disappeared. I tried to place new props in their place just to see if perhaps the game had lost track of them and the new props wanted to stack as if the old one was still there. I placed some just to see what would happen and they 'floated' upwards out of sight. Not quite sure how to reproduce this one, just wanted to mention it.

**To Reproduce**
Unsure.
1. Place props on boxes and other surfaces.
2. build around them
3. check back to see if they're still there?

**Screenshots/Videos/Twitch Clips**

**Additional context**
Sorry if this was less than helpful, but this is a pretty weird one.

**Comment by garybrucey** [comment]
Having the same issue, so thought I'd add on to this. Overall impression, VERY COOL!

**Describe the bug**
I created a castle interior with some props and it seems like the same props keep disappearing after I go to the main menu and back to the campaign. I placed a bear rug in front of a table with a chest on it and some torches around the table. The bear rug is replaced with a copy of the chest but all other props remain. Also, I placed a gold pile in the back corner and put some castle ruin wood in front to block it, then placed a pumpkin and cauldon on the wood. The wood, pumpkin and cauldron disappear each time but the gold remains. 

**To Reproduce**
Steps to reproduce the behavior:
1. Place CastleRuins wood (the 2 block one) and a pumpkin and cauldron on top.
2. Exit to main menu
3. Return to campaign
4. Sad face, as props are gone

**Screenshots/Videos/Twitch Clips**
![image](https://user-images.githubusercontent.com/2657618/79026253-1001db00-7b3d-11ea-8e30-53d3e95cde52.png)!


**Additional context**
And of course this issue fixed itself while I was trying to repro... One thing that did change in between is that I sent the "join link" to some buddies and one of them joined in. Maybe this weird game state was overwritten during that?

While building, I noticed I was drawing some weird cubes on accident. Could those be hide cubes and somehow I placed stuff inside of them? If so, that would be weird because I couldn't see the props even in the GM menu.

![bitmoji](https://sdk.bitmoji.com/render/panel/d8e66e64-ec9c-495f-947e-9bfe52d2523f-327054eb-dd63-4c33-85e7-737f4e420266-v1.png?transparent=1&palette=1&width=246)
